generate_attestation_file.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
```

Comment:
Why not just make this file a ruby script instead of requiring bash?

Reply:
yep that makes sense; will do